src/core/Microsoft.PowerToys.Settings.UI.Lib/IPowerToySettings.cs

```diff
@@ -7,7 +11,9 @@ namespace Microsoft.PowerToys.Settings.UI.Lib
     public interface IPowerToySettings
```

Comment:
nit: Would be good to have xml documentation for `interfaces` to help implement the interfaces.